src/System.Data.SqlClient/src/System/Data/SqlClient/PoolBlockingPeriod.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Nit: Is this line in license needed? The top 3 lines seem to be available in some files that I randomly checked.

Reply:
Some files have this licence line and some don't. For example SortOrder.cs has it.